### ISSUE TITLE
Wrist instability

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10715]
**Preferred term label:**
Wrist instability

**Synonyms**

**Definition (free text, please give PubMed ID)**
Subjective or objective feeling of clunking/clicking in the wrist with movement, wrist 'giving way' or difficulty twisting objects

**Parent term (use hpo.jax.org/app)**
Similar to https://hpo.jax.org/browse/term/HP:0003994, but that should be a sibling term.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Perilunate dislocation (PLD)
Carpal instability nondissociative (CIND)
Scapholunate advanced collapse (SLAC)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0025800, Wrist instability
